libs/scraper-agent/mocked/mockAgent.ts

```
@@ -45,22 +47,13 @@ export class Agent implements AgentClass {
-              type: "navigation",
-              timestamp: i,
-              to: "https://www.example.com",
-            }
-          ]
-        })
-      )
-    };
+      events: this.mockEventsResponse(),
+      accounts: this.mockAccountResponse(),
+    }
+  }
```

ESLint

[error] Missing semicolon.

(semi)